### ISSUE TITLE
Bug/Enchancement

### ISSUE THREAD

**Issue by AniGitHub93** [BartKieviet/Bookkeeper#63]
Very Useful addon, but has the following **bug** for the trading shortcuts on trading screens 
_"https://screenshots.firefox.com/TTQe9mmReJGQEXEr/artemis.pardus.at"_ it displays the value NaN, 

also it will be very helpful if the trade shortcut buttons were customizable similar to the old and now incompatible(with FF> 56 ) **Pardus Co-pilot addon.**

**Comment by BartKieviet** [comment]
That's in Firefox?

Can you tell me if you had a mag scoop open or not? Or stuff packed?

**Comment by AniGitHub93** [comment]
Nope, didn't have a mag scoop installed nor did I have packed stuff..


Regards,
Anirudh Barath.

**Comment by AniGitHub93** [comment]
And yes it's Firefox quantum..

Anirudh Barath

**Comment by BartKieviet** [comment]
I'm giving this one to @valitas  who is our FF expert

**Comment by BartKieviet** [comment]
One more question: did you manually click the food or water *max* link before you hit the button?

**Comment by AniGitHub93** [comment]
I don't know what you mean by the "Max link" but all I do is enter an SB
and press the trade shortcut button that is designed by the add-on "SF E/AE
ENERGY" whilst having food and water on board and I get the text "NaN" in
energy/animal embryo input Boxes.. Pressing it a second time makes the text
disappear and values proportionate to my cargo space appear in the 'energy
animal embryo' input boxes but the values of "food and water" from their
respective input boxes disappear,  forcing me to press the the displayed
value of them in order to make space to buy the energy/Animal embryo..also
the 'NaN text' doesn't appear when Ive no food/water on board.

**Comment by AniGitHub93** [comment]
Hey,

I just signed the petition "Maneka Gandhi: Save Children Who Are Lost
in Kerala Flood 2018" and wanted to see if you could help by adding
your name.

Our goal is to reach 1,901 signatures and we need more support. You
can read more and sign the petition here:

https://chn.ge/2N7cFlo

Thanks!
Anirudh
